### PR TITLE
DevDocs: Update the Section-header readme file

### PR DESCRIPTION
**Pull request by mapk**
Update the Section-header readme file to new layout. See #24282.

### CODE REVIEW

File: client/components/section-header/README.md [hunk 1]
```diff
@@ -31,9 +30,10 @@ const MyHeader = ( { translate } ) => (
...
-This is the base component and acts as a wrapper for a section's (a list of cards) title and any action buttons that act upon that list (like Bulk Edit or Add New Item).
 
 #### Props
-- `className` - *optional* (string|object) Classes to be added to the rendered component.
-- `label` - *optional* (string) The text to be displayed in the header.
+
+Name | Type | Default | Description
+---- | ---- | ---- | ----
+`className` | `string` | null | (Optional) Classes to be added to the rendered component.
+`label` | `string` | null | The text to be displayed in the header.
```
Comment (scruffian) [inline client/components/section-header/README.md]: The default for label is `''` not `null`.  It is technically optional too.

File: client/components/section-header/README.md [hunk 2]
```diff
@@ -31,9 +30,10 @@ const MyHeader = ( { translate } ) => (
...
-## Section Header
-This is the base component and acts as a wrapper for a section's (a list of cards) title and any action buttons that act upon that list (like Bulk Edit or Add New Item).
 
 #### Props
-- `className` - *optional* (string|object) Classes to be added to the rendered component.
-- `label` - *optional* (string) The text to be displayed in the header.
+
+Name | Type | Default | Description
+---- | ---- | ---- | ----
+`className` | `string` | null | (Optional) Classes to be added to the rendered component.
```
Comment (drw158) [inline client/components/section-header/README.md]: Instead of marking which ones are optional, we should mark the required ones with an asterisk.

Comment (mapk) [reply client/components/section-header/README.md]: Is there an example that I can copy? I'm not sure where we should put the asterisk.

Comment (drw158) [reply client/components/section-header/README.md]: you can find an example here: https://github.com/Automattic/wp-calypso/blob/452ee4cacd28b5bbb7fe2d5394fb16271e8be644/docs/component-readme-template.md#props

**Comment by alisterscott** [comment]
Is this PR still in progress or does it require further review?

**Comment by alisterscott** [comment]
Closing due to inactivity - please reopen if necessary

**Pull request closed without merging**